iocBoot/templates/Makefile.base

```diff
@@ -13,6 +13,7 @@ SHELL = /bin/bash
 
```

Comment:
Picking a random line for this to start a thread

Sample size 1 for running this on `lcls-twincat-motion-example`:
- the files created are reasonable
- error messages on make:

```
WARNING:pytmc.parser:Pytmc failed to figure out what to do with: /cds/group/pcds/epics-dev/zlentz/lcls-twincat-motion-example/lcls-twincat-motion-example/_Config/NC/NC.xti
Traceback (most recent call last):
  File "/cds/home/z/zlentz/pydev/pytmc/parser.py", line 2054, in _pre_load_xti_files
    key = get_key(xti_filename, xti)
  File "/cds/home/z/zlentz/pydev/pytmc/parser.py", line 2037, in get_key
    raise ValueError(f'Hmm: {candidates}')
ValueError: Hmm: [(<class 'pytmc.parser.Image'>, '1', 'nc.xti'), (<class 'pytmc.parser.Axis'>, '1', 'nc.xti'), (<class 'pytmc.parser.Axis'>, '2', 'nc.xti'), (<class 'pytmc.parser.Axis'>, '3', 'nc.xti')]
WARNING:pytmc.parser:Pytmc failed to figure out what to do with: /cds/group/pcds/epics-dev/zlentz/lcls-twincat-motion-example/lcls-twincat-motion-example/_Config/PLC/tc_mot_example.xti
Traceback (most recent call last):
  File "/cds/home/z/zlentz/pydev/pytmc/parser.py", line 2054, in _pre_load_xti_files
    key = get_key(xti_filename, xti)
  File "/cds/home/z/zlentz/pydev/pytmc/parser.py", line 2037, in get_key
    raise ValueError(f'Hmm: {candidates}')
ValueError: Hmm: [(<class 'pytmc.parser.Instance'>, '#x08502000', 'tc_mot_example.xti'), (<class 'pytmc.parser.POU'>, '{7b41e58a-08f0-4959-90b4-af08ef5cbf10}', 'tc_mot_example.xti')]
```

Reply:
Current runtime behavior is:
- nothing happens, stalls on failed connections forever (plc is off)

Reply:
I tried digging into those warnings a bit but came out empty-handed. My only guess is that the old project is saved slightly differently compared to our standards. At least it's not fatal...